lib/src/nutribot/nutribot_recommendation_message_component.html

```diff
@@ -12,7 +12,10 @@ <h3>
         <p [innerHtml]="cardDescription"></p>
     </div>
 
-    <widget-button class="buy-button" link [linkHref]="recipe.buyUrl" [externalLink]="true" (click)="logNutribotClickBuy()" [text]="messages.nutribotMessages.buyTreat" *ngIf="!isNutribotLinkExternalEvent && isShortenNutribotJourneyEnabled">
+    <widget-button class="buy-button" link [linkHref]="recipe.buyUrl" [externalLink]="true"
+                   (click)="logNutribotClickBuy()"
+                   [text]="messages.nutribotMessages.buyTreat(recipe.vendorName)"
```

Comment:
this all will change after the backend implementation it will be exactly like you want but i did that for now because the need only that for Walmart not to all clients